app/agent/collector.scala

```diff
@@ -43,7 +46,7 @@ class CollectorAgent[T<:IndexedItem](val collectorSet: CollectorSet[T], sourceSt
     sourceStatusAgent.update(datum.label)
     datum.label match {
       case l@Label(product, origin, size, _, None) =>
-        val marker = Markers.appendEntries((origin.toMarkerMap ++ l.toMarkerMap ++ this.toMarkerMap ++ Map("duration" -> timeSpent)).asJava)
+        val marker = Markers.appendEntries((origin.toMarkerMap ++ l.toMarkerMap ++ this.toMarkerMap ++ Map("CrawlDuration" -> timeSpent)).asJava)
```

Comment:
we've got CrawlDuration here and `duration` in the other place - maybe worth making these field names super explicity

Reply:
Great, thanks Phil! 👍🏽😀

I've added a `durationType` field so that we can keep using the `duration` field where required and add a unique type to make the value of that marker explicit!